### PR TITLE
GitHub check issue command step 1

### PR DESCRIPTION
**Pull request by khouloudbelguith**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | master
| Description?  | Permits to check PrestaShop issues and duplicated issues created from a specific date  
| Category?     | Improvement
| How to test?  | Run this command `php bin/console github:check:issue`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18452)

### CODE REVIEW

**Comment by khouloudbelguith** [comment]
Ping @boubkerbribri @Progi1984

File: src/App/Command/GithubCheckIssueCommand.php [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    {
+        $this->github = new Github($input->getOption('ghtoken'));
+        $time = time();
+
+        $date = new DateTime();
+        $date->sub(new DateInterval('P1D'));
+        $requests = [
+            // Check Issue created and closed
+            //'Created Issue' => 'is:closed created:>='.$date->format('Y-m-d'),
+            'Created Issue' => 'is:issue created:>=2020-03-11',
```
Comment (SimonGrn) [inline src/App/Command/GithubCheckIssueCommand.php:60]: You should calculate this date automatically. If it's 4 weeks in the past, it's easy to do. How do you choose it ?

File: src/App/Command/GithubCheckIssueCommand.php [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->github = new Github($input->getOption('ghtoken'));
+        $time = time();
+
+        $date = new DateTime();
+        $date->sub(new DateInterval('P1D'));
+        $requests = [
+            // Check Issue created and closed
+            //'Created Issue' => 'is:closed created:>='.$date->format('Y-m-d'),
```
Comment (SimonGrn) [inline src/App/Command/GithubCheckIssueCommand.php:59]: You can remove this line.

File: src/App/Command/GithubCheckIssueCommand.php [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                InputOption::VALUE_OPTIONAL,
+                '',
+                $_ENV['GH_TOKEN']
+            )
+            ->addOption(
+                'request',
+                null,
+                InputOption::VALUE_OPTIONAL
+            )
+            ->addOption(
```
Comment (Progi1984) [inline src/App/Command/GithubCheckIssueCommand.php:43]: You can remove this option. I don't think it is useful

**Comment by matks** [comment]
Hello @khouloudbelguith I think this PR is stale 😉  so I close it . Feel free to open another one

**Pull request closed without merging**